app/scripts/controllers/meeting.status.ctrl.js

```diff
@@ -94,8 +94,22 @@ angular.module('dashboard')
             }
         }
 
+        function canMeetingToBeClosed() {
+            $log.debug("meetingStatusCtrl: canMeetingToBeClosed");
+            var result = true;
+            if (angular.isObject(self.mtgDetails) && angular.isArray(self.mtgDetails.topicList)) {
+                self.mtgDetails.topicList.forEach(function (t) {
```

Comment:
Would be nice to break on first match because
1) If only one match is needed then no need to iterate the rest
2) Isn't this a bug: function returns the evaluation result of the last topicList item?